includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -237,46 +262,62 @@ private function getLoggedInUserData( $user ) {
 			'tracking_label' => 'account.sign-out',
 		];
 
+		if ( $this->hasAuthorProfile( $user ) && !empty( $wgEnableAuthorProfileLinks ) ) {
+			$viewProfileLinks[] = [
+				'type' => 'link-text',
+				'href' => $this->getAuthorProfileUrl( $userName ),
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-user-view-author-profile'
+				],
+				'tracking_label' => 'account.profile.author',
```

Comment:
Can we ask him if it's possible to label that follows convention? All account labels are dash separated, maybe it's possible to use it here?